koji-containerbuild.spec

```diff
@@ -1,3 +1,12 @@
+# Enable Python 3 builds for Fedora
+%if 0%{?fedora}
+%bcond_without python3
```

Comment:
I think you're right, this looks backwards.

Reply:
It is right, as it allows you to run 'rpmbuild -bb koji-containerbuild.spec --without python3' even in case python3 is available. But we can remove it, not sure if it makes sense these days (copied from older koji spec).

Reply:
Oh, ok, I wasn't familiar with [conditional builds](http://rpm.org/user_doc/conditional_builds.html). Thanks for clarifying it!